Containers: add mappings for s390x and ppc64le platform architectures to corresponding rids.

tmds opened the pull request:
Rids for these architectures are defined in the rid graph, and Red Hat provides images for them, like [ubi8/dotnet-70-runtime](https://catalog.redhat.com/software/containers/ubi8/dotnet-70-runtime/633c2b3f7f3af1e4ef67fa27).

@baronfel ptal.

tmds committed:
Containers: add mappings for s390x and ppc64le platform architectures to corresponding rids.

baronfel commented:
Can you add this to https://github.com/dotnet/sdk/pull/33166 and extend the tests that I added in that PR instead? I don't want to lose this and in that PR I extracted out the RID mapping to a separate component for reuse as a build task + easier testing.

baronfel commented:
Also, it's cool as heck that you all have such a broad image baseline!

tmds commented:
> Can you add this to https://github.com/dotnet/sdk/pull/33166 and extend the tests that I added in that PR instead? I don't want to lose this and in that PR I extracted out the RID mapping to a separate component for reuse as a build task + easier testing.

Sure, I'll do that once that PR is merged.

tmds commented:
Closing in favor of https://github.com/dotnet/sdk/pull/33166.

tmds commented:
@baronfel depending on when you think https://github.com/dotnet/sdk/pull/33166 will land, would you consider merging this?

baronfel commented:
@tmds yeah we're going to take this now because 8.0.100-p6 snap is today. Should auto-merge once green. Thanks again for adding this :)
